config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -32,6 +32,17 @@ spec:
         spec:
           description: KafkaClusterSpec defines the desired state of KafkaCluster
           properties:
+            alertManagerConfig:
+              description: AlertManagerConfig defines configuration for alert manager
+              properties:
+                maxBrokerCount:
```

Comment:
Should we name it `upScaleLimit` and `downScaleLimit`. Also can we add description here for each field?

`upScaleLimit` - The upper broker count limit the cluster can be upscaled  to.  If the number of brokers is above this limit than auto upscaling triggered by alerts is disabled.

`downScaleLimit` - The lower broker count limit the cluster can be downscaled  to.  If the number of brokers is below this limit than auto downscaling triggered by alerts is disabled.

Reply:
I agree with you. I'll replace them.